Core/Security/CS.py

```diff
@@ -2,10 +2,10 @@
 __RCSID__ = "$Id$"
 
 from DIRAC.ConfigurationSystem.Client.Config                 import gConfig
-from DIRAC.ConfigurationSystem.Client.Helpers.Registry       import getDefaultUserGroup
+from DIRAC.ConfigurationSystem.Client.Helpers.Registry       import getDefaultUserGroup, getVOForGroup
 from DIRAC.ConfigurationSystem.Client.Helpers.Registry       import getDefaultVOMSAttribute, getDefaultVOMSVO, findDefaultGroupForDN
 from DIRAC.ConfigurationSystem.Client.Helpers.Registry       import getUsernameForDN, getGroupsForDN, getHostnameForDN
-from DIRAC.ConfigurationSystem.Client.Helpers.Registry       import getDNForUsername
+from DIRAC.ConfigurationSystem.Client.Helpers.Registry       import getDNForUsername, getUserOption
```

Comment:
Can't this module just be 
`from DIRAC.ConfigurationSystem.Client.Helpers.Registry import *`
? 

BTW I think you should run autopep8 on it.

Reply:
No please, no `*` import. And bwt, it's against pep8 (https://www.python.org/dev/peps/pep-0008/#id23)

Reply:
Yes, it can be. Just that if I get rid of the CS.py it should be verified and too many places. I did not want to make it as part of this PR

Reply:
I have get rid of the CS.py use